### PR TITLE
03-JavaServiceSkeleton 

### PR DESCRIPTION
**Pull request by vad32m**
Implementation of service for transferring float numbers and two apps for using it.

### CODE REVIEW

File: 03-JavaServiceSkeleton/FloatViewer/app/src/main/java/com/example/floatviewer/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        Intent intent = new Intent();
+        intent.setClassName("com.example.floatexchangeservice","com.example.floatexchangeservice.FloatExchangeService");
+
+        try {
+            if (bindService(intent, mConnection, BIND_AUTO_CREATE) == false)
```
Comment (BulldozerBSG) [inline 03-JavaServiceSkeleton/FloatViewer/app/src/main/java/com/example/floatviewer/MainActivity.java:63]: Where is unbindService() ?

File: 03-JavaServiceSkeleton/FloatWriter/app/src/main/java/com/example/floatwriter/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                }
+            }
+
+        });
+
+        Intent intent = new Intent();
+        intent.setClassName("com.example.floatexchangeservice","com.example.floatexchangeservice.FloatExchangeService");
+
+        try {
+            if (bindService(intent, mConnection, BIND_AUTO_CREATE) == false)
```
Comment (BulldozerBSG) [inline 03-JavaServiceSkeleton/FloatWriter/app/src/main/java/com/example/floatwriter/MainActivity.java:63]: Where is unbindService() ?

**Review by BulldozerBSG — CHANGES_REQUESTED** [review]

File: 03-JavaServiceSkeleton/FloatWriter/app/src/main/java/com/example/floatwriter/MainActivity.java (hunk 2 above)
Comment (vad32m) [reply 03-JavaServiceSkeleton/FloatWriter/app/src/main/java/com/example/floatwriter/MainActivity.java:63]: fixed

File: 03-JavaServiceSkeleton/FloatViewer/app/src/main/java/com/example/floatviewer/MainActivity.java (hunk 1 above)
Comment (vad32m) [reply 03-JavaServiceSkeleton/FloatViewer/app/src/main/java/com/example/floatviewer/MainActivity.java:63]: fixed

**Review by BulldozerBSG — APPROVED** [review]

**Pull request merged**